contracts/cw20-stake-external-rewards/src/contract.rs

```diff
@@ -164,6 +196,51 @@ pub fn execute_fund_native(
     }
 }
 
+pub fn execute_withdraw(
+    deps: DepsMut,
+    env: Env,
+    info: MessageInfo,
+) -> Result<Response<Empty>, ContractError> {
+    // only the owner can withdraw
+    let config = CONFIG.load(deps.storage)?;
+    if config.owner != Some(info.sender.clone()) {
+        return Err(Unauthorized {});
```

Comment:
can we make a more specific unauthorized error message here?

Reply:
I kind of followed the pattern I saw on prop-single:
https://github.com/DA0-DA0/dao-contracts/blob/693a5d6a36ada1505c5a52e8029b1b80ea557393/contracts/cw-proposal-single/src/contract.rs#L339-L343

No strong preference though, what do others think?